.gitignore

```diff
@@ -4,3 +4,4 @@ dist/*
 /.cabal-sandbox
 /cabal.sandbox.config
 cloud-remote.pdf
+/.stack-work/
```

Comment:
I think you should put this in the global gitignore, but given that using stack is almost certainly quite common, and we already have the same for cabal sandboxes, this can stay.